### ISSUE TITLE
Reports - Scatter plot displayed but aggregate data shows "Data Load Error"

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#410]
This doesn't happen all the time. With small, well behaved data sets the aggregate data is displayed. This case is for real city data with approx 1700 records and a lot of PM fields. I will try to create an anonymized data set that shows this behavior.

**Comment by RDmitchell** [comment]
When trying to graph Site EUI vs Gross Floor area, the scatter plot works but the bar chart doesn't. 

![image](https://cloud.githubusercontent.com/assets/6314950/10205586/e089dd16-6776-11e5-91c3-b582a2e8905e.png)

I get this error message

https://seedtest.lbl.gov/app/get_aggregated_building_report_data/?end_date=…nization_id=19&start_date=2013-01-01&x_var=site_eui&y_var=gross_floor_area 

Failed to load resource: the server responded with a status of 500 (INTERNAL SERVER ERROR)

#BuildingReportsController: Error loading agg chart data : undefined  angular.js:9509 

![image](https://cloud.githubusercontent.com/assets/6314950/10205400/d929a7a0-6775-11e5-8063-82e8df5a3d3e.png)

But it works if I select Site EUI vs Year Built -- if it had to do with field mapping, it seems like neither plot would have data. So it must be something else. I will try to put together a simpler set of data that shows this problem 

![image](https://cloud.githubusercontent.com/assets/6314950/10205524/78da2e0a-6776-11e5-92a8-cdfc130d2962.png)

**Comment by cycustodio** [comment]
Is it because "Year Built" and "Site EUI" are both from portfolio-manager and "Gross Floor Area" is in covered-buildings?

**Comment by RDmitchell** [comment]
It does seem to be related to floor area. 

I have put a sample set of data (5 records in the Issue 410 gdrive folder)
https://drive.google.com/a/lbl.gov/folderview?id=0B3fTKpZ9Dx7LMFp5Qk94cVlOMkE&usp=sharing

All the graphs work except for bar charts with anything plotted against Gross Floor Area on the Y-axis.

![image](https://cloud.githubusercontent.com/assets/6314950/10207219/e94130d6-677f-11e5-9295-570eb73fbbf7.png)

To Claudine's question, both the covered building and the PM file have a field mapped to Gross Floor Area

**Covered Building Mapping**

![image](https://cloud.githubusercontent.com/assets/6314950/10207129/65c456a2-677f-11e5-944f-5fc3b6618219.png)

**PM Data mapping**

![image](https://cloud.githubusercontent.com/assets/6314950/10207170/9f604560-677f-11e5-8a83-6b9326e7d23a.png)

**Comment by StephenCzarnecki** [comment]
For starters when I do the default matching I see it matched 2 of the 5 buildings.  But the graph behavior is as you say so it is picking up the 5 in the PM file.

When I look at the code for the aggregate buildings it first calls my code and gets a list of the correct building snapshot ids.  Later on there is a bit about grouping the floor areas and there is one record that has a floor area that is causing an error in the lookup.  I've changed it so all floor areas > 1m sq feet goes in the "> 1,000 k" bin.

Checked in a fix and made a pull request:  https://github.com/SEED-platform/seed/pull/413
